common/validity/language.xml

```diff
@@ -623,33 +625,33 @@
 			mis mul
 			zxx
 		</id>
-		<id type='language' idStatus='deprecated'>		<!-- 283 items -->
-			aam adp agp ais ajt~u als aoh arb asd aue ayr ayx~y azj
+		<id type='language' idStatus='deprecated'>		<!-- 296 items -->
+			aam adp agp ais ajp ajt~u als aoh arb asd aue ayr ayx~y azj
 			baz bbz bcc bcl bgm bh bhk bic bij bjd bjq bkb blg bmy bpb btb btl bxk bxr bxx byy
 			cbe cbh cca ccq cdg cjr cka cld cmk cmn cnr coy cqu cug cum cwd
-			daf dap dgo dgu dha dhd dik diq dit djl dkl drh drr drw dud duj dwl dzd
+			daf dap dgo dgu dha dhd dik diq dit djl dkl drh drr drw dud duj dwl
```

Comment:
"dzd" was removed here

Web search for "dzd deprecated" turns up this file:

https://www.iana.org/assignments/lang-subtags-templates/dzd-2023-03-17.txt

which reads as follows:

FOR ARCHIVING: Registration form for 'dzd'

---

LANGUAGE SUBTAG REGISTRATION FORM

1. Name of requester: Doug Ewell
2. E-mail address of requester: doug at ewellic.org

3. Record Requested:

Type: language
Subtag: dzd
Description: Daza

4. Intended meaning of the subtag:

5. Reference to published description of the language (book or article):

6. Any other relevant information:

This registration tracks a change made to ISO 639-3 effective
2023-01-20, adding the code element 'dzd' for Daza, which had been
retired in 2015 as non-existent. The net effect of this registration is
to remove the Deprecated value from this record.

For more information on the ISO 639-3 change, refer to:
https://iso639-3.sil.org/request/2022-027